roc_bike_growth/metrics.py

```diff
@@ -0,0 +1,70 @@
+from collections import deque
+
+# Resiliency metric
+# Taken from https://codereview.stackexchange.com/questions/184392/computing-resilience-of-the-network-presented-as-an-undirected-graph-in-python
+def resilience(graph, attack_order):
```

Comment:
How is `attack_order` determined? Would a default k and random selection of nodes make sense?

Reply:
Also, does it make sense to add an example into one of the notebooks (or add a new notebook?). I know we're not using formal tests, but that may help at least by showing expected behavior and a clear use case that can be maintained.